sdk/daml-lf/tests/ReplayBenchmark.daml

```diff
@@ -0,0 +1,17 @@
+-- Copyright (c) 2025 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module ReplayBenchmark where
```

Comment:
Is `daml-lf/tests` the best place for this Daml file?

Reply:
On [Add a non-standard flag within Canton to enable this snapshot dumping functionality](https://github.com/digital-asset/daml/pull/21601) I'll be adding in some Daml script code. I'll review these specifics then.